src/main/java/io/zeebe/monitor/ZeebeSimpleMonitorApp.java

```diff
@@ -58,6 +64,20 @@ public Executor asyncExecutor() {
   @Bean
   public Mustache.Compiler configureFallbackValueForMissingVariablesInMustacheTemplates(
       Mustache.TemplateLoader templateLoader) {
-    return Mustache.compiler().defaultValue("⍰").withLoader(templateLoader);
+    return Mustache.compiler().defaultValue("Ôì░").withLoader(templateLoader);
```

Comment:
@schmetzyannick one more minor thing, this UTF-8 character is broken.
The pom.xml states, this project uses UTF-8, so please adjust your editor the next time.
I will fix this afterward.